utils/subpath.go

```diff
@@ -148,6 +148,12 @@ func UpdateAssetsSubpathFromConfig(config *model.Config) error {
 		return nil
 	}
 
+	// Similarly, don't rewrite during a CI build, when the assets may not even be present.
+	if os.Getenv("IS_CI") == "true" {
+		mlog.Debug("Skipping update to assets subpath since CI build")
+		return nil
```

Comment:
The error in question is logged higher up in the stack, but here we explicitly avoid it by returning `nil`.